components/clm/bld/namelist_files/use_cases/1950_CMIP6HR_control.xml

```diff
@@ -18,7 +18,6 @@
 <stream_year_last_popdens  phys="clm4_5" use_cn=".true." ndepsrc="stream" >1955</stream_year_last_popdens>
 
 <!-- CMIP6 HighResMIP compsets -->
-<fsurdat hgrid="ne120np4" >lnd/clm2/surfdata_map/surfdata_ne120np4_simyr1950_c20180108.nc </fsurdat>
```

Comment:
@bishtgautam , why do some compsets still have finidat defined in the use_case xml file? I thought your plan was to move them all to the defaults xml file.

Reply:
I only made changes for fsurdat + finidat for F1850SC5-CMIP6, F2010C5-CMIP6-LR, and F2010C5-CMIP6-HR. While change for fsurdat is straightforward, the finidat is a little complicated. If you provide me with res+compset for all CMIP6 compsets that are supported in maint-1.0, I will make relevant changes.

Reply:
I am not sure about other CMIP6 F-compsets, but there are what I used on maint-1.0:
F1850SC5-CMIP6.ne30
F2010C5-CMIP6-LR.ne30
F2010C5-CMIP6-HR.ne120

I made the comment above on the 1950 file, but it is also true for 1850_CMIP6_control.xml and 1850_SCMIP6_control.xml (one of which is used by F1850SC5-CMIP6) that contain the finidat setting. Is there a reason why it's not moved to the default file? As long as there is a reason for that (not overlooked), I am fine with the changes.